src/Raven.Studio/typescript/viewmodels/database/tasks/editReplicationHubTask.ts

```diff
@@ -0,0 +1,480 @@
+        this.editedReplicationAccessItem(cloneItem);
+        this.initTooltips();
+    }
+
+    warnAboutUnsavedChanges() {
+        return this.confirmationMessage("Unsaved changes",
+            `<div>You have unsaved changes. How do you want to proceed?</div>`, {
+                buttons: ["Cancel", "Continue"],
+                html: true
+            })
+    }
+  
+    deleteReplicationAccessItem(accessItemToDelete: replicationAccessHubModel) {
+        this.confirmationMessage("Are you sure?", 
+            `<div>Delete Replication Access: ${accessItemToDelete.replicationAccessName()}?</div>`, {
+            buttons: ["Cancel", "Delete"],
+            html: true
+        })
+            .done(result => {
+                if (result.can) {
+                    const self = this;
+                    new deleteReplicationHubAccessConfigCommand(this.activeDatabase(), 
+                        this.editedHubTask().taskName(), accessItemToDelete.certificate().thumbprint())
+                        .execute()
+                        .done(() => {
+                            new getReplicationHubAccessCommand(self.activeDatabase(), self.editedHubTask().taskName())
+                                .execute()
+                                .done((accessResult: Raven.Client.Documents.Operations.Replication.ReplicationHubAccessResult) => {
+                                    self.processResults(accessResult);
+                                });
+                        })
+                }
+            });
+    }
+
+    private goToOngoingTasksView() {
+        router.navigate(appUrl.forOngoingTasks(this.activeDatabase()));
+    }
+    
+    private initTooltips() {
+        this.setupDisableReasons();
+        
+        popoverUtils.longWithHover($("#hub-to-sink-info"),
+            {
+                content:
+                    "<ul class='margin-bottom margin-bottom-xs padding'>" +
+                        "<li><small>These prefixes define what docments are allowed to be <strong>sent from the Hub.</strong></small></li>" +
+                        "<li><small>You can <strong>further restrict this list</strong> when defining a Sink task that receives data from this Hub.</small></li>" +
+                    "</ul>"
+            });
+
+        popoverUtils.longWithHover($("#sink-to-hub-info"),
+            {
+                content:
+                    "<ul class='margin-bottom margin-bottom-xs padding'>" +
+                        "<li><small>These prefixes define what docments are allowed to be <strong>sent to this Hub.</strong></small></li>" +
+                        "<li><small>You can <strong>further restrict this list</strong> when defining a Sink task that sends data to this Hub.</small></li>" +
+                    "</ul>"
+            });
+    }
+
+    importCertificate(fileInput: HTMLInputElement) {
+        this.spinners.importCertificate(true);
+        fileImporter.readAsText(fileInput,data => this.certificateImported(data));
```

Comment:
space after comma

Reply:
if read as text is sync then no need to spinners

Reply:
ok - got it - done